client/src/art/addSvg.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const AddSvg = props => (
+const AddSvg = (props: {}) => (
```

Comment:
I think there might already be a React type that we can use here that something like "element attributes", but I'm good with this as a starting point :)

Reply:
Yeah, not sure if there is something like that. We can change it later if any